packages/react-component-library/src/components/TopLevelNavigation/SearchBar/partials/StyledSearchBar.tsx

```diff
@@ -1,21 +1,23 @@
 import styled from 'styled-components'
 import { selectors } from '@defencedigital/design-tokens'
 
+import { StyledInput } from '../../../TextInput/partials/StyledInput'
+
 const { zIndex, mq, fontSize } = selectors
 
 export const StyledSearchBar = styled.div`
-  z-index: ${zIndex('dropdown', 1)}
+  z-index: ${zIndex('dropdown', 1)};
   display: block;
   margin-top: -1px;
   position: absolute;
   width: 100%;
 
-  .rn-textinput__input {
+  ${StyledInput} {
     font-size: ${fontSize('l')};
   }
 
   ${mq({ gte: 'xs' })`
-    .rn-textinput__input {
+    ${StyledInput} {
       font-size: ${fontSize('base')};
     }
   `}
```

Comment:
I've fixed this, but I find this a bit weird if I'm being honest (especially as it's the only media query in the Masthead).

Reply:
I think we should remove it.

Reply:
Yep, cool. I'll just delete these then and leave it at the default 16px (which is how it's been for ages).